src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

```diff
@@ -515,16 +515,16 @@ class Project: ADOSVTBase
                     $controlResult.AddMessage([VerificationResult]::Passed,"Number of administrators configured are within than the approved limit: $($this.ControlSettings.Project.MaxPAMembersPermissible).");
                 }
 
-                if($TotalPAMembers -gt 0){
-                    $controlResult.AddMessage("Count of Project Administrators: $($TotalPAMembers)")
+                #if($TotalPAMembers -gt 0){
```

Comment:
Why this condition is removed?

Reply:
same already checked above and this line comes under that